### PR TITLE
Adopt CoC Covenant

### PR DESCRIPTION
**Pull request by hynek**
As discussed in https://mail.python.org/pipermail/cryptography-dev/2016-March/000581.html ff.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -5,5 +5,6 @@ recursive-include   doc         *
 recursive-include   examples    *
 recursive-include   rpm         *
 recursive-exclude   leakcheck   *.py *.pem
+recursive-exclude   examples    *.crt *.key *.cert *.pkey
```
Comment (hawkowl) [inline MANIFEST.in]: this seems like an unrelated change ;)

**Comment by hawkowl** [comment]
+1 for Contributor Covenant (it's what Twisted uses, so there's a plenty good precedent for picking this one above others)

File: MANIFEST.in (hunk 1 above)
Comment (hynek) [inline MANIFEST.in]: yep! ;)  but it prevents the tests to fail under certain circumstances (some I am in right now).  I can remove it tho to not spoil this special moment. ;)

**Comment by hawkowl** [comment]
LGTM.

**Comment by hynek** [comment]
Use your powers then.

**Comment by hawkowl** [comment]
Drat, I got in to work an hour late ;)
